src/TypeScript.UnitTests/EslintBridgeClient/EslintBridgeClientTests.cs

```diff
@@ -145,6 +147,22 @@ public void Dispose_DisposesHttpWrapper()
             httpWrapper.VerifyNoOtherCalls();
         }
 
+        [TestMethod]
+        public void Dispose_FailsToCallClose_HttpWrapperStillDisposed()
```

Comment:
Test for the try-catch in the dispose (probably should've split it to another PR... my bad).